source/sdk/lang/Exception.ooc

```diff
@@ -197,13 +197,13 @@ OSException: class extends Exception {
 }
 
 OutOfBoundsException: class extends Exception {
-	init: func (=origin, accessOffset, elementLength: SizeT) {
+	init: func (=origin, accessOffset, elementLength: Int) {
 		init(accessOffset, elementLength)
 	}
-	init: func ~noOrigin (accessOffset, elementLength: SizeT) {
-		message = "Trying to access an element at offset %d, but size is only %d!" format(accessOffset, elementLength)
+	init: func ~noOrigin (accessOffset, elementLength: Int) {
+		message = "Trying to access an element at offset %i, but size is %i!" format(accessOffset, elementLength)
```

Comment:
"%d scans an integer as a signed decimal number, but %i defaults to decimal but also allows hexadecimal (if preceded by "0x") and octal if preceded by "0"."
I learned something today, thanks.